test_unstructured_ingest/test-ingest-local.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+SCRIPT_DIR=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &>/dev/null && pwd)
+cd "$SCRIPT_DIR"/.. || exit 1
+
+PYTHONPATH=. ./unstructured/ingest/main.py \
+    --metadata-exclude filename \
+    --local-input-path ./example-docs/ \
+    --local-file-glob layout-parser-paper-fast.* \
```

Comment:
nit: let's do *.html as it should be faster, and also more inline with a typical glob pattern.
that should also be in quotes or bash will try not to expand it and also be clear as such, so:

```
    --local-file-glob "*.html"
```